lte/protos/mconfig/mconfigs.proto

```diff
@@ -170,7 +170,10 @@ message PipelineD {
         string hmac_key = 7;
     }
     HEConfig he_config = 17;
-
+    message Enable5GFeatureSet{
+        bool enable = 1;
+    }
```

Comment:
Do we intend to add more fields to this message in the future?

Reply:
Yes, "node_identifier" can be added in future, as I can see in pipelined.yml file.
![image](https://user-images.githubusercontent.com/70897856/128728720-676fa529-9b9f-44f5-9fb5-7bdc7ad5fb36.png)